.circleci/config.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       - attach_workspace:
           at: ~/gluestick
 
-      - run: yarn run test --coverage
+      - run: yarn run test -i --coverage
```

Comment:
Generally I prefer the non-aliased version of flags when it exists in a file: `--runInBand`.